CHANGELOG

```diff
@@ -120,6 +120,18 @@ New and updated examples and miniapps
 - Added a modification of ex9 in examples/hiop that demonstrates the nonlinear
   constrained optimization interface and the use of the SLBQP and HiOp solvers.
 
+Improved testing
+----------------
+- Added a new directory, tests/scripts, with several shell scripts that perform
+  simple checks on the code including: code styling, documentation formatting,
+  proper use of .gitignore, and preventing the accidental commit of large files.
+
+- It is recommended that the tests scripts above are run by the developers via
```

Comment:
"It is recommended that the tests scripts above _be_ run by the developers"

May sound weird, but is correct as far as I can tell. @acfisher would you agree?

Reply:
Will reword